scripts/pi-hole/js/index.js

```diff
@@ -686,7 +686,8 @@ function updateTopLists() {
         }
 
         domain = escapeHtml(domain);
-        url = '<a href="queries.php?domain=' + domain + '">' + domain + "</a>";
+        urlText = domain === "" ? "." : domain;
```

Comment:
Maybe try this here:

```suggestion
        urlText = domain ? domain : ".";
```